applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -508,13 +489,80 @@ static void on_gnss_evt_pvt(void)
 	}
 }
 
+static int do_cloud_send_msg(const char *message, int len)
+{
+	int err;
+	struct nrf_cloud_tx_data msg = {
+		.data.ptr = message,
+		.data.len = len,
+		.topic_type = NRF_CLOUD_TOPIC_MESSAGE,
+		.qos = MQTT_QOS_0_AT_MOST_ONCE
+	};
+
+	err = nrf_cloud_send(&msg);
+	if (err) {
+		LOG_ERR("nrf_cloud_send failed, error: %d", err);
+	}
+
+	return err;
+}
+
+static void send_location(struct nrf_modem_gnss_nmea_data_frame * const nmea_data)
+{
+	if (!nrf_cloud_ready || !location_signify || !nmea_data) {
+		return;
+	}
```

Comment:
Can this be checked at the position like before?
Also nmea_data should not be NULL as this function is called in "} else {"